backend/src/main/java/com/bakdata/conquery/resources/admin/rest/AdminDatasetResource.java

```diff
@@ -94,13 +101,36 @@ public void addTable(@FormDataParam("table_schema") FormDataBodyPart schemas) th
 	@POST
 	@Path("imports")
 	public void addImport(@QueryParam("file") File selectedFile) throws IOException, JSONException {
-		if(!selectedFile.canRead() || !selectedFile.exists() || !selectedFile.isAbsolute() || !selectedFile.getPath().endsWith(ConqueryConstants.EXTENSION_DESCRIPTION)) {
-			throw new WebApplicationException("Invalid file (`" + selectedFile + "`) specified: Needs to be absolute path, readable and be a .cqpp-file.", Status.BAD_REQUEST);
-		}
+
+		if(!selectedFile.exists())
```

Comment:
Das hattes du ja schon anders implementiert in einem anderen PR oder?

Reply:
jo, wohl ausversehen gepushed!